pvacseq/lib/main.py

```diff
@@ -229,6 +227,8 @@ def main(args_input = sys.argv[1:]):
             class_i_alleles.append(allele)
         if allele in MHCII.all_valid_allele_names():
             class_ii_alleles.append(allele)
+        if allele not in MHCI.all_valid_allele_names() and allele not in MHCII.all_valid_allele_names():
```

Comment:
The dictionary used to create the list returned by `all_valid_allele_names` is stored on the class so the long-ish work of parsing the files that contains this information only gets done once. However, I think you are right and we should avoid checking them again. I will restructure these conditionals.